bin/cij_plotter

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python
```

Comment:
Is it on purpose that this is implemented in a single file, unlike the runner/reporter/analyser?

Reply:
Yeah, I just wanted something simple and self-contained.
In a second iteration, it should be part of the module code, where the plotter/data-set transformation are dynamically loadable from packages/module similar to the loading of extractors.

Reply:
Sounds good